text_recognizer/data/iam_synthetic_paragraphs.py

```diff
@@ -29,6 +30,7 @@ class IAMSyntheticParagraphs(IAMParagraphs):
     def __init__(self, args: argparse.Namespace = None):
         super().__init__(args)
         self.trainval_transform.scale_factor = 1  # we perform rescaling ahead of time, in prepare_data
+        self.transform.scale_factor = 1
```

Comment:
Not having this line is the bug discussed in the below comment.

Reply:
- [ ] I still need to handle resizing cleanly. I have cleaned up the IAM image loading and inversion logic.